custom_components/tahoma/climate_devices/atlantic_electrical_heater.py

```diff
@@ -52,35 +52,31 @@ def supported_features(self) -> int:
     @property
     def hvac_mode(self) -> str:
         """Return hvac operation ie. heat, cool mode."""
-        return HVAC_MODE_OFF if self.preset_mode == PRESET_NONE else HVAC_MODE_HEAT
+        return TAHOMA_TO_HVAC_MODES[self.select_state(*CORE_ON_OFF_STATE)]
 
     @property
     def hvac_modes(self) -> List[str]:
         """Return the list of available hvac operation modes."""
-        return [*MAP_HVAC_MODES]
+        return [*HVAC_MODES_TO_TAHOMA]
+
+    async def async_set_hvac_mode(self, hvac_mode: str) -> None:
+        """Set new target hvac mode."""
+        await self.async_execute_command(
+            COMMAND_SET_HEATING_LEVEL, HVAC_MODES_TO_TAHOMA[hvac_mode]
```

Comment:
This command won't work for the PASS_ACTUATOR device.

Here are the commands sent by the Cozytouch application : 

Set to Off (the command is named "off" without parameters): 
```
2021-06-01 10:22:50 INFO (MainThread) [custom_components.tahoma] 2021-06-01 10:21:12: off executed via Android 2.5.4#244 Set heating level in room c423fb05-92c5-49b4-a917-c7ac553e7ef4 on io://xxxx-xxxx-xxxx/2634956, with [].
```

Set to On (there is no command for "on", it's just set the preset to "comfort"): 
```
2021-06-01 10:22:50 INFO (MainThread) [custom_components.tahoma] 2021-06-01 10:21:37: setHeatingLevel executed via Android 2.5.4#244 Set heating level in room c423fb05-92c5-49b4-a917-c7ac553e7ef4 on io://xxxx-xxxx-xxxx/2634956, with ['comfort'].
```

Reply:
Off should work already in this PR, but I understood that I have to rename `on` to `comfort`?

Reply:
I just test it and yes, it's working for "off" (it's not the same command sent by the cozytouch app but it doesn't matter).

It will work ok with "comfort", but you need to keep the "on" value somewhere for the "hvac_mode(self)" function.